site/app/templates/polls/ViewPollResults.twig

```diff
@@ -0,0 +1,31 @@
+<script>
+    window.onload = function() {
+        dataPoints = [];
+        {% for answer, number in results %}
+            string = "{{ poll.getResponseString(answer}}"
+            dataPoints.push({ y: {{ number }}, label: string}),
+        {% endfor %}
+        title = "{{ poll.getName() }}"
+        console.log(dataPoints);
+        var chart = new CanvasJS.Chart("chartContainer", {
+            animationEnabled: true,
+            theme: "light2", // "light1", "light2", "dark1", "dark2"
+            title:{
+                text: title
+            },
+            axisY: {
+                title: "Response Count"
+            },
+            data: [{        
+                type: "column",
+                dataPoints: dataPoints
+            }]
+        });
+        chart.render();
+    }
+</script>
+
+<div class="content">
+    <div id="chartContainer" style="height: 370px; width: 100%;"></div>
+    <script src="https://canvasjs.com/assets/script/canvasjs.min.js"></script>
```

Comment:
I dont think this is how you are supposed to be adding dependencies https://submitty.org/developer/updating_dependencies